applications/ShapeOptimizationApplication/python_scripts/custom_ios/wrl_io.py

```diff
@@ -0,0 +1,74 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Geiser Armin, https://github.com/armingeiser
+#
+# ==============================================================================
+
+import KratosMultiphysics as KM
+from .wrl_reader import read_shapes
+
+
+def _rename_to_valid_name(model_part, shape):
```

Comment:
It could. but then codacy will complain that no members are used and it could be a function.
It is declared "private" because it should only be used within this module.